doc/submitting.md

```diff
@@ -0,0 +1,27 @@
+## Submitting changes to SUSE Build Service
+
+Submissions are managed by a SUSE internal jenkins node in the InstallTools tab.
+
+Each time a new commit is integrated into the master branch of the repository,
+a new submit request is created in the SUSE Build Service.
+
+For maintained branches, the package is submitted to a devel project but the final submission
+must be triggered manually.
+
+`*.changes` and version numbers are auto-generated from git commits, you don't have to worry about this.
+
+The spec file is maintained in the Build Service only. If you need to change it, submit to the devel project
+in the build service directly. The current names of the devel projects can be seen in the jenkins logs.
```

Comment:
This is a bit tricky as the Jenkins is internal only. That means the external contributors have no chance to see them. Is there any other possibility?